garage/envs/mujoco/randomization/__init__.py

```diff
@@ -1,6 +1,4 @@
 from garage.envs.mujoco.randomization.randomized_env import randomize
-from garage.envs.mujoco.randomization.variation import Distribution
-from garage.envs.mujoco.randomization.variation import Method
```

Comment:
I think these are okay. It means that Distribution and Method are both part of the public interface to `garage.envs.mujoco.randomization`.